src/ImportPage.js

```diff
@@ -19,27 +22,92 @@ type Props = {|
 |};
 
 export default function ImportPage({onDataImported}: Props) {
+  const processTimeline = useCallback(
+    (events: TimelineEvent[]) => {
+      // Filter null entries and sort by timestamp.
+      // I would not expect to have to do either of this,
+      // but some of the data being passed in requires it.
+      events = events.filter(Boolean).sort((a, b) => (a.ts > b.ts ? 1 : -1));
+
+      if (events.length > 0) {
+        const processedData = preprocessData(events);
+        const processedFlamechart = preprocessFlamechart(events);
+        onDataImported(processedData, processedFlamechart);
+      }
+    },
+    [onDataImported],
+  );
+
+  // ImportPage is show in production build only
```

Comment:
This comment isn't very accurate as ImportPage is still shown in dev.


```suggestion
  // DEV only: auto-import a demo profile on component mount (i.e. page load)
```